business/services.go

```diff
@@ -275,6 +276,26 @@ func (in *SvcService) GetService(namespace, service, interval string, queryTime
 	return &s, nil
 }
 
+func (in *SvcService) UpdateService(namespace, service string, interval string, queryTime time.Time, jsonPatch string) (*models.ServiceDetails, error) {
```

Comment:
Note, in the current reduced scenario, Kiali UI is not updated the annotations.
Users typically can update directly the controller.

We can leave this method in the code as it's similar as the workload one, but edition of the annotations are out of the scope for the moment.